mass_mailing_custom_unsubscribe/models/mail_unsubscription.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from openerp import _, api, fields, models
+from .. import exceptions
+
+
+class MailUnsubscription(models.Model):
+    _name = "mail.unsubscription"
+    _inherit = "mail.thread"
+    _rec_name = "date"
+
+    date = fields.Datetime(
+        default=lambda self: self._default_date(),
```

Comment:
Well, that guideline is outdated (no new API), because you can overwrite the field only the default definition to get a new default, but OK, until we change it, follow the other comment.